client/components/plans/plan-actions/index.jsx

```diff
@@ -225,7 +234,9 @@ const PlanActions = React.createClass( {
 			const link = purchasesPaths.managePurchase( this.props.site.slug, this.props.sitePlan.id );
 
 			return (
-				<a href={ link } className="button plan-actions__upgrade-button">{ this.translate( 'Manage Plan', { context: 'Link to current plan from /plans/' } ) }</a>
+				<a href={ link } className="button plan-actions__upgrade-button">{
+						this.translate( 'Manage Plan', { context: 'Link to current plan from /plans/' } )
+				}</a>
```

Comment:
Do you mean providing the `<a>` tag as a component? In this case since we don't have mixed html/strings here I think the current usage is ok.